src/modules/accounting/accounting.py

```diff
@@ -93,6 +96,16 @@ def is_extra_data_submitted(self, blockstamp: BlockStamp) -> bool:
         processing_state = self._get_processing_state(blockstamp)
         return processing_state.extra_data_submitted
 
+    def check_sanity(self, blockstamp: ReferenceBlockStamp) -> bool:
+        if ALLOW_NEGATIVE_REBASE_REPORTING:
+            return True
+        cl_rebase_report = self.simulate_cl_rebase(blockstamp)
+        frame_cl_rebase = self.bunker_service.get_cl_rebase_for_current_report(blockstamp, cl_rebase_report)
+        if frame_cl_rebase < 0:
+            logger.info({'msg': 'CL rebase is negative.', 'value': frame_cl_rebase})
```

Comment:
I would suggest showing this message as a warning on several lines and even if `ALLOW_NEGATIVE_REBASE_REPORTING` is turned on